test/qa-tests/jstests/stat/stat_custom_headers.js

```diff
@@ -20,36 +20,36 @@
   x = runMongoProgram.apply(this, ["mongostat", "--port", port,
     "-o", "host,conn,time", "-n", 4, "--humanReadable=false"].concat(commonToolArgs));
   assert.eq(x, 0, "mongostat should succeed with -o and -n options");
-  assert.eq.soon(5, function() {
+  assert.eq.soon(6, function() {
     rows = allShellRows();
     return rows.length;
-  }, "expected 5 rows in mongostat output");
-  assert.eq(statFields(rows[0]).join(), "host,conn,time",
-    "first row doesn't match 'host conn time'");
-  assert.eq(statFields(rows[1]).length, 3,
+  }, "expected 6 rows in mongostat output");
+  assert.eq(statFields(rows[1]).join(), "host,conn,time",
+    "second row doesn't match 'host conn time'");
+  assert.eq(statFields(rows[2]).length, 3,
     "there should be exactly three entries for a row of this stat output");
   clearRawMongoProgramOutput();
 
   // basic -o
   x = runMongoProgram.apply(this, ["mongostat", "--port", port,
     "-o", "host,conn,time", "-n", 4].concat(commonToolArgs));
   assert.eq(x, 0, "mongostat should succeed with -o and -n options");
-  assert.eq.soon(5, function() {
+  assert.eq.soon(6, function() {
     rows = allShellRows();
     return rows.length;
-  }, "expected 5 rows in mongostat output");
-  assert.eq(statFields(rows[0]).join(), "host,conn,time",
-    "first row doesn't match 'host conn time'");
-  assert.eq(statFields(rows[1]).length, 5,
-    "there should be exactly five entries for a row of this stat output (time counts as three)");
+  }, "expected 6 rows in mongostat output");
+  assert.eq(statFields(rows[1]).join(), "host,conn,time",
+    "second row doesn't match 'host conn time'");
+  assert.eq(statFields(rows[2]).length, 5,
+    "third row should be exactly five entries for a row of this stat output (time counts as three)");
```

Comment:
```suggestion
    "third row should have exactly five entries (time counts as three)");
```
this sentence didn't totally make sense as written